wonnx/templates/pool/reduce.wgsl

```diff
@@ -79,6 +81,11 @@ fn main(@builtin(global_invocation_id) global_id: vec3<u32>) {
 					else if(accumulator < input_val) {
 						accumulator = input_val;
 					}
+				{% elif op_type == "ArgMax" %}
+					if(input_val > max_element) {
+                        max_element = input_val;
+                        accumulator = f32(count);
```

Comment:
The output of `ArgMax` is actually specified as `int64` (see [here](https://onnx.ai/onnx/operators/onnx__ArgMax.html)). This should work for now but we might have to fix it later.

Reply:
Good point, then it would probably make sense to move argmax out of reduce to not have to return mixed data types from reduce?